kview/src/main/java/dev/ikm/komet/kview/mvvm/view/properties/AddFullyQualifiedNameController.java

```diff
@@ -136,6 +139,12 @@ public void updateView() {
         populate(statusComboBox, fqnViewModel.findAllStatuses(getViewProperties()));
         populate(caseSignificanceComboBox, fqnViewModel.findAllCaseSignificants(getViewProperties()));
         populate(languageComboBox, fqnViewModel.findAllLanguages(getViewProperties()));
+
+        // Set UI to default values
+        caseSignificanceComboBox.setValue(Entity.getFast(TinkarTerm.DESCRIPTION_NOT_CASE_SENSITIVE.nid()));
```

Comment:
Can you try not to use Entity.getFast()? just use `TinkarTerm.DESCRIPTION_NOT_CASE_SENSITIVE` instead.
UI controls should really hold `EntityFacade` ComboBox<EntityFacade>. not entity objects from the database fetch.

Reply:
@carldea done, changed the ComboBox type that was there in the previous code to EntityFacade.
Couldn't stop using getFast for the StatusComboBox because State is an enum and the only thing I can get there is its nid.
Good catch!